frontend/src/components/NewAgreement/AgreementCreatedSuccessfully.tsx

```diff
@@ -1,41 +1,84 @@
-import { Box, Button, Container, Divider, Stack, Typography } from '@mui/material';
+import { Button, Container, Divider, IconButton, Stack, SvgIcon, Typography } from '@mui/material';
 import { FC, useContext } from 'react';
 import { useTranslation } from 'react-i18next';
-import DoneRoundedIcon from '@mui/icons-material/DoneRounded';
 import { StringBank } from 'strings';
 import { useNavigate } from 'react-router-dom';
 import { GroupContext } from 'contexts/group';
+import handLogo from 'assets/icons/hand-v@3x.png';
+import styled from 'styled-components';
+import { ReactComponent as CopyIcon } from 'assets/icons/copy.svg';
 
-const AgreementCreatedSuccessfully: FC = () => {
+const CopyIconWrapper = styled(SvgIcon)`
+  svg path {
+    fill: #adb2b8;
+  }
+`;
+
+const CopyEL: FC<{ text: string }> = ({ text }) => {
+  return (
+    <IconButton
+      sx={{ marginLeft: '2rem', marginRight: '1rem' }}
+      onClick={() => {
+        navigator.clipboard.writeText(text);
+      }}
+    >
+      <CopyIconWrapper>
+        <CopyIcon />
+      </CopyIconWrapper>
+    </IconButton>
+  );
+};
+
+const AgreementCreatedSuccessfully: FC<{ agreementId: string }> = ({ agreementId }) => {
   const { t } = useTranslation();
   const { slug: groupSlug } = useContext(GroupContext);
   const navigate = useNavigate();
+  const AgreementText = `${window.location.origin}/${groupSlug}/agreement/${agreementId}`;
   return (
     <Container maxWidth="xs" sx={{ height: '100%' }}>
       <Stack alignItems="center" justifyContent="center" paddingX={6} spacing={2} height="100%">
-        <DoneRoundedIcon
-          fontSize="large"
-          sx={{ bgcolor: 'success.main', borderRadius: '50%', padding: 0.5 }}
-        />
+        <img width="165px" height="180px" src={handLogo}></img>
         <Typography variant="h2" textAlign="center">
           {t(StringBank.AGREEMENT_PUBLISHED_SUCCESSFULLY)}
         </Typography>
-        <Box />
-        <Button variant="contained" size="large" color="primary" fullWidth>
+        <Button variant="contained" size="large" sx={{ backgroundColor: '#8d54ea' }} fullWidth>
           {t(StringBank.VIEW_AGREEMENT)}
         </Button>
         <Button
           variant="contained"
           size="large"
           fullWidth
+          sx={{
+            marginBottom: '0.5rem !important',
+          }}
           onClick={() => navigate(`/${groupSlug}/active-agreements`)}
         >
           {t(StringBank.RETURN_TO_ALL_AGREEMENTS)}
         </Button>
-        <Divider flexItem sx={{ paddingY: 1 }} />
+        <Divider flexItem sx={{ marginTop: '2rem' }} />
```

Comment:
In general it's better practice to use symmetric margins, because it retains similar behavior on various screen sizes

Reply:
removed, since the stack style overrides it anyaway